### PR TITLE
eth-bridge-integration: sync with 0.13.0

### PR DESCRIPTION
**Pull request by juped**
The thing to look out for here is the faucet POW integration, which touches everything.

Incorporates #1002 for some easy conflict resolution, even though that's properly its own change.

### CODE REVIEW

File: apps/src/lib/node/ledger/shell/prepare_proposal.rs [hunk 1]
```diff
@@ -130,10 +130,13 @@ where
...
             .iter()
-            .map(|tx| {
-                Tx::from(match tx.decrypt(privkey) {
-                    Ok(tx) => DecryptedTx::Decrypted(tx),
-                    _ => DecryptedTx::Undecryptable(tx.clone()),
+            .map(|queued_tx| {
+                Tx::from(match queued_tx.tx.decrypt(privkey) {
+                    Ok(decrypted_tx) => DecryptedTx::Decrypted {
+                        tx: decrypted_tx,
+                        has_valid_pow: queued_tx.has_valid_pow,
```
Comment (batconjurer) [inline apps/src/lib/node/ledger/shell/prepare_proposal.rs:137]: Doesn't this need a feature flag over the `has_valid_pow` field?

File: shared/src/ledger/queries/router.rs [hunk 2]
```diff
@@ -250,7 +250,7 @@ macro_rules! try_match_segments {
         $end = $request.path.len();
         match $request.path[$start..$end].parse::<$arg_ty>() {
             Ok(parsed) => {
-                println!("Parsed {}", parsed);
+                // println!("Parsed {}", parsed);
```
Comment (batconjurer) [inline shared/src/ledger/queries/router.rs:253]: What happened here?

**Review by batconjurer — CHANGES_REQUESTED** [review]

File: shared/src/ledger/queries/router.rs (hunk 2 above)
Comment (batconjurer) [reply shared/src/ledger/queries/router.rs:253]: caveman debugging :hammer_and_pick:

**Comment by batconjurer** [comment]
> is this still relevant?

No, we merge it a later version from main already.

**Comment by adrianbrink** [comment]
> > is this still relevant?
> 
> No, we merge it a later version from main already.

Closing it then

**Pull request closed without merging**